monai/fl/client/monai_algo.py

```diff
@@ -359,6 +359,7 @@ def __init__(
         eval_workflow_name: str = "train",
         train_workflow: BundleWorkflow | None = None,
         eval_workflow: BundleWorkflow | None = None,
+        stats_sender: Callable | None = None,
```

Comment:
Why is this needed if you can get stats_sender from "extra" in "initialize"?
Why is "stats_sender" callable here?

Reply:
This was from the original PR. I do not know why we need it anymore, so I can remove it from the init args.